packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -941,6 +941,14 @@ watch(remoteParameterOptionsLoading, () => {
 	tempValue.value = displayValue.value as string;
 });
 
+// Focus input field when changing from fixed value to expression
+watch(isModelValueExpression, async (isExpression, wasExpression) => {
```

Comment:
can you please add tests?

Reply:
I'm not sure how to test the focused state of the inline expression input. It's using Codemirror; hence, it's rendered as a div, not an input